scripts/upgradeConfig.ts

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+import { readdir, readFile, writeFile } from 'fs-extra';
+import simpleGit from 'simple-git';
+
+/**
+ * Script: upgradeConfigs.ts
+ * Run with: ts-node scripts/upgradeConfig.ts
+ * ------------------------------------------
+ * Upgrades the references to \@solid/community-server in
+ * package.json (lsd prefix) and all JSON-LD config files.
+ * This script is run alongside standard-version
+ * after the version bump is done in package.json but before the
+ * release has been committed.
+ */
+
+/**
+ * Helper function to escape search strings for use in Regular Expressions
+ * @param  string - the search string to be escaped
+ * @returns Promise with the escaped string
+ */
+async function escapeString(string: string): Promise<string> {
+  return string.replace(/[$()*+.?[\\\]^{|}]/gu, '\\$&');
+}
+
+interface NpmPackage {
+  version: string;
+  name: string;
+}
+/**
```

Comment:
```suggestion
}

/**
```
And a few more places below.